lib/symboldatabase.cpp

```diff
@@ -737,6 +737,33 @@ void SymbolDatabase::createSymbolDatabaseFindAllScopes()
                 } else {
                     tok = tok->link();
                 }
+            } else if (Token::Match(tok, "%name% (")) {
+                if (tok->next() && Token::simpleMatch(tok->next()->link(), ") ;")) {
```

Comment:
I think the `tok->next() &&` is a bit overkill here. We check on the line above that it is a "(" token.

Instead of `tok->next()->link()` I suggest `tok->linkAt(1)` .